community-tutorials/debootstrap/01-en.md

```diff
@@ -0,0 +1,311 @@
+
+## Step 2.3 - (Copy SSH information)
+
+Since you likely use SSH to connect to your machine, you would want to be able to root into it again. Also if your machine is allowed to SSH into other servers, you would like to retain that possibility.
+
+```bash
+cp -a /root/.ssh/ /mnt/debinst/root/
+```
+
+Also make sure that your SSH configuration is present for the SSH server
+```bash
+cp -a /etc/ssh/sshd_conf.d/* /mnt/debinst/etc/ssh/sshd_conf.d/
+```
+This only works if you have already put your configuration into the sshd_conf.d directory. If it still resides in the `sshd_conf` file, you might either extract the information (recommended) or copy that file.
+
+To avoid problems regarding the host_key identification, we backup the host keys which where generated when we installed ssh and copy the host keys from the old system to the new system.
+```bash
+zip -R /mnt/debinst/root/old_hostkeys_bak.zip /mnt/debinst/etc/ssh/ssh_host*
+rm /mnt/debinst/etc/ssh/ssh_host*
+cp -a /etc/ssh/ssh_host* /mnt/debinst/etc/ssh/
+```
+
+## Step 2.4 - (Copy network information)
+
+Like with the SSH configuration, we update the network configuration as well.
+```bash
+cp -a /etc/network/interfaces.d/* /mnt/debinst/etc/network/interfaces.d/
+```
+Again if your configuration resides in /etc/network/interfaces, you might consider migrating.
+
+**Be aware that [Debian changed its network interface names from version 10 to 11](https://wiki.debian.org/NetworkInterfaceNames)** and therefore you might run into serious trouble. 
+
+An approach to avoid this is to use the network-manager on first boot and only after then edit the interface definition, once you know the correct naming:
+```bash
+apt install network-manager
+```
+
+## Step 2.5 - (Set hostname)
+```bash
+cp /etc/hostname /mnt/debinst/etc/hostname
+```
+Or choose a new hostname if you want to
+
+## Step 2.6 - (Mount/Copy other directories)
+
+For any other directory (e.g. /home) you might want to consider a copy or a mount inside (/etc/fstab) from a different partition.
+Also copy all required config files and data for other services that run on your machine like apache or mysql.
+
+# Step 3 - (Finalize your setup)
+
+With all config files in place it is time to finalize the installation. If you're not inside your new system chroot to it again:
+
+```bash
+chroot /mnt/debinst /bin/bash
+```
+
+The next steps are pretty much the same as for a normal Debian setup
+
+## Step 3.1 - (Set timezone)
+```bash
+dpkg-reconfigure tzdata
+```
+
+## Step 3.2 - (Set local)
+```bash
+apt install -y locales
+dpkg-reconfigure locales
+```
+
+## Step 3.3 - (Set keyboard layout)
+```bash
+apt install -y console-setup console-setup-linux
+dpkg-reconfigure locales
+```
+
+## Step 3.4 - (Set root password)
+```bash
+passwd
+```
+
+## Step 3.5 - (Install kernel and bootloader)
+
+Now let's make it bootable. This step does not (yet) modify your current setup:
+```bash
+linux-image-amd64 firmware-linux grub2
```

Comment:
apt-get install is missing

Reply:
I prever apt :P But you are correct we need at least one of them